network/src/peer/mod.rs

```diff
@@ -140,7 +153,7 @@ where
 
     pub async fn start(mut self) {
         let remote_peer_id = self.remote_peer_id();
-        info!(
+        trace!(
```

Comment:
I can see value in changing info to trace, but I'm wondering the logic behind changing the info to trace?  Do we have a large number of logs coming from our testing?

Also, we should put this into a separate commit around improving the logging.

Reply:
I don't think these per-actor init/stop messages are buying us enough. especially for public-facing nodes, emitting 10+ logs every time someone connects to us seems a bit crazy to me

Reply:
Could you separate out the refactor bits from the fuzzing of this commit into a separate refactor?  Looks like they're completely file separate, so it shouldn't be too bad.

Reply:
done!